ui/src/environments/test-env-base.ts

```diff
@@ -12,4 +12,5 @@ export const testEnvironmentBase = {
   zendeskHelpCenterUrl: 'http://aousupporthelp.zendesk.com/hc',
   shibbolethUrl: 'http://mock-nih.dev.test.firecloud.org',
   shouldShowDisplayTag: true,
+  enableHomepageRestyle: true
```

Comment:
not sure if this is required here

Reply:
seems like it essentially duplicates ui/src/environments/environment.test.ts and ui/src/environments/environment.local.ts (and gets overwritten by those files anyway)

Reply:
The idea behind this file was that you can define env variables here, and not redefine them in the other files, to clarify exactly how each environment differs from the base "test" environment. If you want to follow that pattern, just remove the redefinition of this value from environment.ts and environment.test.ts

(We could also follow-up at some point with environment.local to have it "inherit" from the test-env-base, since it should share most of the same properties as the test env.)

Reply:
So I'm a little confused but my understanding is that `src/environments/environment.ts` gets replaced anyway in all environments except dev. CH and I were discussing in https://pmi-engteam.slack.com/archives/CHRN2R51N/p1567115237109500

That means that the env variable in `environment.ts` can be removed, as you also mentioned.

I think we wanted this feature flag to be enabled for the test environment, which would mean leaving it in `environment.test.ts`. Does that sound right/reasonable?

Reply:
Yes, environment.ts gets replaced wholesale depending on which environment is being built & deployed.

But if you look at environment.test.ts, all it does it splat out the variables from test-env-base.ts (see this line https://github.com/all-of-us/workbench/blob/master/ui/src/environments/environment.test.ts#L5) and then add any overrides necessary. I created this pattern in an attempt to minimize the needless duplication between our test-aligned environment configs, & so it's easier to clearly tell which variables are different from the "base" test config.

I'm not 100% sure my attempt has succeeded, but it should be an easy pattern to follow, e.g. if you leave "enableHomepageRestyle: true" in the test-env-base and don't overwrite it in either the local or test config files.

Reply:
Got it. Thanks for explaining!